packages/core/src/styles/layout/docs/grid.stories.mdx

```diff
@@ -12,6 +12,15 @@ layout. Spacing, alignments and gap between items can all be configured with
 the component. The grid layout by default is a **12** column grid.
 Gap values are t-shirt size `[xxs, xs, sm, md, lg, xl, xxl]` and align with the [Layout Spacing Tokens](./?path=/docs/experimental-spacing-getting-started--page).
 
+<cds-alert-group status="warning" cds-layout="m-b:lg">
+  <cds-alert>
+    The grid utility is still a work in progress for IE11 support. For responsive multi column layouts in IE11 use{' '}
+    <a cds-text="link" href="https://clarity.design/documentation/grid">
+      Clarity CSS Grid.
+    </a>
+  </cds-alert>
+</cds-alert-group>
+
```

Comment:
Until I get a reliable IE testbed working and know exactly what parts post CSS supports I am calling this out explicitly.